### PR TITLE
updates to supported platforms and portfolio

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/about_certified_platforms.mdx [hunk 1]
```diff
@@ -1,48 +1,54 @@
...
 
 export const certifiedPlatforms = {
-  lanner7573a : {
-    application : "Medium/Large Branch",
-    cpu : "8C (Atom)",
-    mem : "16GB",
+  lenovosr650r2 : {
+    application : "Large Datacenter, 100G hub",
+    cpu : "22C (Xeon Gold)",
+    mem : "128-192GB",
```
Comment (MichaelBaj) [inline docs/about_certified_platforms.mdx:10]: A range for memory is strange. If our min requirements are 128GB, we should state 128GB. A customer can always add more memory if required. If the min required is 192, we should state as such.

File: docs/about_certified_platforms.mdx [hunk 2]
```diff
@@ -1,48 +1,54 @@
 ---
-title: Certified Plus Platforms
-sidebar_label: Certified Plus Platforms
+title: Certified Portfolio
+sidebar_label: Certified Portfolio
 ---
 
 export const certifiedPlatforms = {
```
Comment (MichaelBaj) [inline docs/about_certified_platforms.mdx:6]: Can we get the storage details added to these tables as well?

File: docs/about_certified_platforms.mdx [hunk 3]
```diff
@@ -51,14 +57,58 @@ export const certifiedPlatforms = {
...
+  },
+  lannernca1515b : {
+    application : "Small/Medium Branch",
+    cpu : "4C (Atom C3558)",
+    mem : "8GB ECC",
+    interfaces : [
+      "Ethernet, 6 RJ45 ports, 2 SFP",
+    ]
+  },
+    lannernca1515a : {
```
Comment (MichaelBaj) [inline docs/about_certified_platforms.mdx:86]: Extra leading spacing

File: docs/about_certified_platforms.mdx [hunk 4]
```diff
@@ -51,14 +57,58 @@ export const certifiedPlatforms = {
...
+    application : "Small/Medium Branch",
+    cpu : "4C (Atom C3558)",
+    mem : "8GB ECC",
+    interfaces : [
+      "Ethernet, 6 RJ45 ports, 2 SFP",
+      "LTE (AT&T, Verizon) Dual SIM"
     ]
-  }
+  },
+    lannernca1515alte : {
```
Comment (MichaelBaj) [inline docs/about_certified_platforms.mdx:103]: Extra leading spacing

File: docs/about_certified_platforms.mdx [hunk 5]
```diff
@@ -84,48 +134,95 @@ export const Platform = ({id}) => (
   </table>
 )
 
-Juniper maintains platforms which have been certified to run the Session Smart Router (SSR) software. For more details on certification, see the [platform support policy](about_supported_platforms.md#ssr-certification-program).
+Juniper maintains platforms which have been certified to run the Session Smart Router (SSR) software. For more details on certification, see the [Platform Support Policy](about_supported_platforms.md#ssr-certification-program).
+
+## Certified Platforms - Small and Medium Branch
+
+The following devices are Certified Platforms for small to medium deployments. Models vary out of the box. Only the SKUs listed are Certified.
```
Comment (MichaelBaj) [inline docs/about_certified_platforms.mdx:141]: "Models vary out of the box." is a strange statement.
Recommended rewording:

The following devices are Certified Platforms for small to medium deployments. Only the SKUs listed are Certified. The platform model alone does not guarantee conformance to software requirements.

File: docs/about_certified_platforms.mdx [hunk 6]
```diff
@@ -84,48 +134,95 @@ export const Platform = ({id}) => (
...
+<Platform id="lannernca1515b"></Platform>
+
+#### L-1515B-4C-8E-128M-C1-SSR
+<Platform id="lannernca1515blte"></Platform>
 
-## Certified Plus
+## Certified Platforms – Medium and Large Branch
 
-The following platforms have the status of [Certified Plus](about_supported_platforms.md#ssr-certified-plus-platform).
+The following devices are Certified Platforms for medium to large deployments. Models vary out of the box. Only the SKUs listed are Certified.
```
Comment (MichaelBaj) [inline docs/about_certified_platforms.mdx:184]: Recommended rewording:

The following devices are Certified Platforms for medium to large deployments. Only the SKUs listed are Certified. The platform model alone does not guarantee conformance to software requirements.

File: docs/about_supported_platforms.md [hunk 7]
```diff
@@ -2,75 +2,55 @@
...
 
 Inclusion in the SSR Certification Program is based on a given platforms ability to continuously meet certain test and support criteria. New platforms are added to the program at Juniper's discretion. For details on how to get new platforms in to the SSR certification program, or further information regarding the program, please contact Juniper support.
 
-### SSR Certified Plus Platform
+### SSR Certified Platform
 
-The SSR Certification program provides a designation of Certified Plus for certain platforms. Certified Plus platforms have the following characteristics:
+The SSR Certification program provides a designation of Certified for certain platforms. CCertified platforms within the portfolio have the following characteristics:
```
Comment (MichaelBaj) [inline docs/about_supported_platforms.md:26]: Typo: Certified

File: docs/about_supported_platforms.md [hunk 8]
```diff
@@ -120,6 +100,8 @@ Platforms may have a variety of different features and capabilities configurable
 * Power profile: maximum performance
 * System version, release date, manufacturer's part number, and serial number set in DMI table
 
+--->
```
Comment (MichaelBaj) [inline docs/about_supported_platforms.md:103]: Why is this arrow here?

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — DISMISSED** [review]

**Review by MichaelBaj — DISMISSED** [review]

File: docs/intro_system_reqs.md [hunk 9]
```diff
@@ -3,26 +3,63 @@ title: System Requirements
 sidebar_label: System Requirements
 ---
 ## Hardware Requirements
+
 The Session Smart Networking Platform runs on both bare metal servers and as a virtual machine within hypervisor environments. For virtual environments the same CPU, memory, and storage specifications are required for comparable throughput.
```
Comment (avijitvjuniper) [inline docs/intro_system_reqs.md:7]: > For virtual environments the same CPU, memory, and storage specifications are required for comparable throughput.

I don't think this is true. We should rephrase this and remove the comparable throughput part

**Review by MichaelBaj — DISMISSED** [review]

**Review by avijitvjuniper — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**